### PR TITLE
Update to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by rivalavocado**

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @rivalavocado Can you please resolve conflicts? Thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,23 +26,23 @@ Before sending your pull requests, make sure you do the following:
...
-  sufficient description, if applicable unit tests are added, if it is a
+  quality requirements. For example, we check if the CLA is signed, if the PR has
+  sufficient description, if applicable unit tests are added, and if it is a
   reasonable contribution (meaning it is not a single liner cosmetic PR).
 
 **2. Valid?**
 
 - If the PR passes all the quality checks then we go ahead and assign a
   reviewer.
 - If the PR didn't meet the validation criteria, we request for additional
```
Comment (vijayasarathib) [inline CONTRIBUTING.md:37]: could you modify this "doesn't" instead of past tense "didn't".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,23 +26,23 @@ Before sending your pull requests, make sure you do the following:
...
-  changes to be made to PR to pass quality checks and send it back or on a rare
+  changes to be made to the PR to pass quality checks and send it back, or on a rare
   occassion we may reject it.
 
 **3. Review**
 
-- For Valid PR, reviewer (person familiar with the code/functionality) checks if
+- For valid PR, reviewer (person familiar with the code/functionality) checks if
```
Comment (vijayasarathib) [inline CONTRIBUTING.md:43]: Valid should be in lower case

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -188,7 +190,7 @@ diff <my_cc_file> /tmp/my_cc_file.cc
 #### Python coding style
 
 Changes to TensorFlow Python code should conform to
```
Comment (vijayasarathib) [inline CONTRIBUTING.md:192]: should be Changes to TensorFlow's Python code should conform to

**Review by vijayasarathib — COMMENTED** [review]
Hello,
I've reviewed the changes made in the CONTRIBUTING.md file and have some grammatical suggestions to enhance clarity and correctness. I have left the comme
1.  Original: "If the PR didn't meet the validation criteria..."
   Suggestion: Use present tense "doesn't" instead of past tense "didn't".
2.Original: "For Valid PR, reviewer..."
   Suggestion: "Valid" should be in lowercase and be "a valid".
 3. Original:Changes to TensorFlow Python code should conform to
    Suggestion: Changes to TensorFlow's Python code should conform to

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -15,7 +15,9 @@ Before sending your pull requests, make sure you do the following:
 
 ## How to become a contributor and submit your own code
 
-![Screen Shot 2022-08-30 at 7 27 04 PM](https://user-images.githubusercontent.com/42785357/187579207-9924eb32-da31-47bb-99f9-d8bf1aa238ad.png)
+![guidelines1](https://github.com/rivalavocado/tensorflow/assets/56943827/3fea1abf-e5bb-4747-be72-8f6449868b84)
```
Comment (mihaimaruseac) [inline CONTRIBUTING.md:18]: Why change the image link?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -15,7 +15,9 @@ Before sending your pull requests, make sure you do the following:
...
+
+[What is G3?](https://github.com/Xtra-Computing/G3)
```
Comment (mihaimaruseac) [inline CONTRIBUTING.md:20]: What is this link doing here?

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by gbaned** [comment]
Hi @rivalavocado Can you please check @mihaimaruseac's comments and resolve conflicts?. Thank you!

**Comment by gbaned** [comment]
Hi @rivalavocado Any update on this PR? Please. Thank you!

**Comment by gbaned** [comment]
Hi @rivalavocado  I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen! Thank you for your contribution!

**Pull request closed without merging**